using-webhooks/client/ios-swift/app/CheckoutViewController.swift

```diff
@@ -0,0 +1,140 @@
+        return button
+    }()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .white
+        let stackView = UIStackView(arrangedSubviews: [cardTextField, payButton])
+        stackView.axis = .vertical
+        stackView.spacing = 20
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        view.addSubview(stackView)
+        NSLayoutConstraint.activate([
+            stackView.leftAnchor.constraint(equalToSystemSpacingAfter: view.leftAnchor, multiplier: 2),
+            view.rightAnchor.constraint(equalToSystemSpacingAfter: stackView.rightAnchor, multiplier: 2),
+            stackView.topAnchor.constraint(equalToSystemSpacingBelow: view.topAnchor, multiplier: 2),
+        ])
+        loadPage()
+    }
+
+    func displayAlert(title: String, message: String, restartDemo: Bool = false) {
+        DispatchQueue.main.async {
+            let alert = UIAlertController(title: title, message: message, preferredStyle: .alert)
+            if restartDemo {
+                alert.addAction(UIAlertAction(title: "Restart demo", style: .cancel) { _ in
+                    self.cardTextField.clear()
+                    self.loadPage()
+                })
+            }
+            else {
+                alert.addAction(UIAlertAction(title: "OK", style: .cancel))
+            }
+            self.present(alert, animated: true, completion: nil)
+        }
+    }
+
+    func loadPage() {
+        // Create a PaymentIntent by calling the sample server's /create-payment-intent endpoint.
+        let url = URL(string: BackendUrl + "create-payment-intent")!
+        let json: [String: Any] = [
+            "currency": "usd",
+            "items": [
+                "id": "photo_subscription"
+            ]
+        ]
+        var request = URLRequest(url: url)
+        request.httpMethod = "POST"
+        request.setValue("application/json", forHTTPHeaderField: "Content-Type")
+        request.httpBody = try? JSONSerialization.data(withJSONObject: json)
+        let task = URLSession.shared.dataTask(with: request, completionHandler: { [weak self] (data, response, error) in
+            guard let response = response as? HTTPURLResponse,
+                response.statusCode == 200,
+                let data = data,
+                let json = try? JSONSerialization.jsonObject(with: data, options: []) as? [String : Any],
+                let clientSecret = json["clientSecret"] as? String,
+                let stripePublicKey = json["publicKey"] as? String else {
+                    let message = error?.localizedDescription ?? "Failed to decode response from server."
+                    self?.displayAlert(title: "Error loading page", message: message)
+                    return
+            }
+            print("Created PaymentIntent")
+            self?.paymentIntentClientSecret = clientSecret
+            Stripe.setDefaultPublishableKey(stripePublicKey)
```

Comment:
```suggestion
            // Configure the SDK with your publishable key so it can make API requests to Stripe
            Stripe.setDefaultPublishableKey(stripePublicKey)
```